### PR TITLE
Add support of the view permissions from Django 2.1

### PR DESCRIPTION
**Pull request by ihabhussein**
Updated to handle the new [model view permissions](https://docs.djangoproject.com/en/2.1/releases/2.1/#model-view-permission) introduced in Django 2.1.

### CODE REVIEW

**Comment by RamezIssac** [comment]
Thank you !
About the view permission (which is only on 2.1) 
How about making this an option/setting (ie support_view_permission) which default to true if django version is 2.1 ?

**Comment by ihabhussein** [comment]
Commit 3d4705e adds check in the code for versions of Django that supports view permissions an displays the widget appropriately. It always shows the `view' column when supported and never when not supported.
Do you think it should be a setting instead?

File: tabular_permissions/templates/tabular_permissions/admin/tabular_permissions.html [hunk 1]
```diff
@@ -6,6 +6,14 @@
     <tr>
         <th> {% trans 'application'|capfirst %}</th>
         <th>{% trans 'model'|capfirst %}</th>
+        <th>
```
Comment (RamezIssac) [inline tabular_permissions/templates/tabular_permissions/admin/tabular_permissions.html]: I think we need the version check (view permission availability check) here

**Comment by ihabhussein** [comment]
The test was on the wrong branch. Sorry about that.
The current comment is d48c6dc

**Comment by RamezIssac** [comment]
Merged and in V 2.2 released on Pypi

**Pull request merged**